system/include/libc/endian.h

```diff
@@ -22,19 +22,19 @@
 
 #include <stdint.h>
 
-static __inline uint16_t __bswap16(uint16_t __x)
+__attribute__((unused)) static __inline uint16_t __bswap16(uint16_t __x)
```

Comment:
we shouldn't modify musl code here, unless we absolutely must (and then we should mark it as diverging from upstream).

Reply:
Those warnings are very annoying, perhaps the changes could be suggested to musl?

Reply:
Agreed, this is worth bringing up with musl. I'm not sure what their policy is on attribute things like that (depends on compiler support, and I'm not sure which musl supports).